skfda/exploratory/visualization/_outliergram.py

```diff
@@ -50,22 +63,39 @@ class Outliergram:
     def __init__(
         self,
         fdata: FDataGrid,
+        *,
+        chart: Union[Figure, Axes, None] = None,
```

Comment:
The only reason for having a `chart` parameter is to be positional only. If it does not make sense for it to be positional, maybe it can be removed (and the code will be simpler).

Reply:
i can move it over the *